src/CSAccounting.sol

```diff
@@ -331,15 +331,18 @@ contract CSAccounting is
     }
 
     /// @inheritdoc ICSAccounting
-    function penalize(uint256 nodeOperatorId, uint256 amount) external onlyCSM {
+    function penalize(
+        uint256 nodeOperatorId,
+        uint256 amount
+    ) external onlyRole(PENALIZE_ROLE) {
         CSBondCore._burn(nodeOperatorId, amount);
     }
 
     /// @inheritdoc ICSAccounting
     function chargeFee(
         uint256 nodeOperatorId,
         uint256 amount
-    ) external onlyCSM {
+    ) external onlyRole(PENALIZE_ROLE) {
```

Comment:
Should we have a separate role here? `CHARGE_ROLE`

Reply:
I don't see any clear reason for it RN. With EIP-7002 funcs charge method will also be required.